### ISSUE TITLE
There was an issue submitting your final solution

### ISSUE THREAD

**Issue by xanderoku** [codewars/codewars.com#1797]
It happened to me already twice: I start a kata, run sample tests as usual, run 'attempt' which is green, then when i press "submit final" i get an error message that there was a problem submitting. So the kata is passed, i see my solution in my profile but when i go to solutions under the kata and press 'my solutions' there is nothing there ('There are no solutions yet for this kata.'). If i pass my solution again nothing changes - my nickname doesn't appear in the list of solutions.

Kinda same as: https://github.com/Codewars/codewars.com/issues/391

Steps to reproduce the behavior (not sure how to exactly reproduce it, issue 391 also has a reproduce howto):
1. Go to 'https://www.codewars.com/kata/57274562c8dcebe77e001012'
2. Click on 'Submit final'
//3. Scroll down to '....'
4. See error 'There was an issue submitting your final solution'

- Same, logged with Github
- OS: Win10
- Browser: Chrome 
- Browser Version: 73.0.3683.103 (64 bit)

**Comment by FArekkusu** [comment]
This is a very common problem. Press `Attempt` again.

**Comment by xanderoku** [comment]
> This is a very common problem. Press `Attempt` again.

That was the first thing I tried to do before posting the issue, but it didn't change anything. I see solutions of other people but not mine. As I said earlier when I press 'My Solutions' below those two katas there is nothing except 'There are no solutions yet for this kata' line. But I see my solutions in 'Profile - Solutions'. Tried now to submit the same solution and a different one - they still don't appear among others.
